tsl/test/expected/dist_compression.out

```diff
@@ -1474,6 +1474,260 @@ SELECT * FROM metric where medium is not null ORDER BY time LIMIT 1;
+_timescaledb_internal._dist_hyper_11_46_chunk
+(3 rows)
+
+
+NOTICE:  [db_dist_compression_3]:  SELECT * from show_chunks('compressed')
+NOTICE:  [db_dist_compression_3]:
+show_chunks                                  
+---------------------------------------------
+_timescaledb_internal._dist_hyper_11_44_chunk
+_timescaledb_internal._dist_hyper_11_45_chunk
+_timescaledb_internal._dist_hyper_11_46_chunk
+(3 rows)
+
+
+ remote_exec 
+-------------
+ 
+(1 row)
+
+SELECT chunk_schema || '.' ||  chunk_name, data_nodes
+FROM timescaledb_information.chunks
+WHERE hypertable_name = 'compressed';
+                   ?column?                    |                             data_nodes                              
+-----------------------------------------------+---------------------------------------------------------------------
+ _timescaledb_internal._dist_hyper_11_44_chunk | {db_dist_compression_1,db_dist_compression_2,db_dist_compression_3}
+ _timescaledb_internal._dist_hyper_11_45_chunk | {db_dist_compression_1,db_dist_compression_2,db_dist_compression_3}
+ _timescaledb_internal._dist_hyper_11_46_chunk | {db_dist_compression_1,db_dist_compression_2,db_dist_compression_3}
+(3 rows)
+
+SELECT count(*) FROM compressed;
+ count 
+-------
+    49
+(1 row)
+
+-- make data node unavailable
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER
+SELECT alter_data_node(:'DATA_NODE_1', port => 55433, available => false);
+WARNING:  could not switch data node on 5 chunks
+                         alter_data_node                         
+-----------------------------------------------------------------
+ (db_dist_compression_1,localhost,55433,db_dist_compression_1,f)
+(1 row)
+
+SET ROLE :ROLE_1;
+-- update compressed chunks
+INSERT INTO compressed SELECT t, (abs(timestamp_hash(t::timestamp)) % 10) + 1, random()*80
+FROM generate_series('2018-03-02 1:00'::TIMESTAMPTZ, '2018-03-04 1:00', '1 hour') t;
+-- chunks on data node 1 are no longer present
+SELECT chunk_schema || '.' ||  chunk_name, data_nodes
+FROM timescaledb_information.chunks
+WHERE hypertable_name = 'compressed';
+                   ?column?                    |                  data_nodes                   
+-----------------------------------------------+-----------------------------------------------
+ _timescaledb_internal._dist_hyper_11_44_chunk | {db_dist_compression_2,db_dist_compression_3}
+ _timescaledb_internal._dist_hyper_11_45_chunk | {db_dist_compression_2,db_dist_compression_3}
+ _timescaledb_internal._dist_hyper_11_46_chunk | {db_dist_compression_2,db_dist_compression_3}
+(3 rows)
+
+SELECT * from show_chunks('compressed');
+                  show_chunks                  
+-----------------------------------------------
+ _timescaledb_internal._dist_hyper_11_44_chunk
+ _timescaledb_internal._dist_hyper_11_45_chunk
+ _timescaledb_internal._dist_hyper_11_46_chunk
+(3 rows)
+
+SELECT * FROM test.remote_exec(ARRAY[:'DATA_NODE_2', :'DATA_NODE_3'], $$ SELECT * from show_chunks('compressed'); $$);
```

Comment:
Not sure what this output is supposed to show. Just showing the chunks again on "available" nodes?

I guess it would be more interesting to see things on the "unavailable" node....